app/controllers/team/projects_controller.rb

```diff
@@ -19,7 +19,11 @@ def unassigned
   end
 
   def handed_over
-    authorize Project, :handed_over?
+    begin
+      authorize Project, :handed_over?
+    rescue Pundit::NotAuthorizedError
+      return head(:not_found)
```

Comment:
I've not seen this `head` before, something about this makes me wonder if there was another approach, but this works - so let's use it.